src/main/java/frc/robot/subsystems/Wrist.java

```diff
@@ -0,0 +1,185 @@
+    // The kWristEncoderZeroTick is the absolute encoders tick value at 0deg
+    // The 10-bit encoder has 1024 ticks and we will use 1024 for +'ve values and 1024 for -'ve values
+    return ((ticks - RobotMap.kWristEncoderZeroTick) / 1024.0) * 180.0;
+  }
+
+  /****************************************************************************************************************************** 
+  **
+  ******************************************************************************************************************************/
+  private double getAngle(int encoderPositionTicks) {
+    return sensorTicksToDegrees(encoderPositionTicks);
+  }
+
+  /****************************************************************************************************************************** 
+  **
+  ******************************************************************************************************************************/
+  private int getEncoderPositionTicks() {
+    // return mMaster.getSensorCollection().getAnalogInRaw();
+    return mMaster.getSelectedSensorPosition(RobotMap.kPIDLoopIdx);
+
+  }
+
+  public void zeroEncoder() {
+    mMaster.setSelectedSensorPosition(0, RobotMap.kPIDLoopIdx, RobotMap.kLongCANTimeoutMs);    
+  }
+
+  /****************************************************************************************************************************** 
+  **
+  ******************************************************************************************************************************/
+  public void MotionMagicOutput(double targetPositionTicks) {
+    if (mControlState != controlMode.kMotionMagic) {
+      mControlState = controlMode.kMotionMagic;
+      mMaster.selectProfileSlot(RobotMap.kMotionMagicSlotIdx, RobotMap.kPIDLoopIdx);
+    }
+    mEncoderPositionTicks = getEncoderPositionTicks();
+    if (mEncoderPositionTicks < targetPositionTicks) {
+      mWristFFGravityComponent = Math.cos(Math.toRadians(getAngle(mEncoderPositionTicks))) * RobotMap.kWristFFGravityComponent;
+    } else {
+      mWristFFGravityComponent = 0.0;
+    }
+    mLogger.info("Encoder position: {}, target position: {}, Feed Forward: {}", mEncoderPositionTicks, targetPositionTicks, mWristFFGravityComponent);
+    mMaster.set(ControlMode.MotionMagic, targetPositionTicks, DemandType.ArbitraryFeedForward, mWristFFGravityComponent);
+  }
+
+  public void setOpenOutput(double zWrist) {
+    mMaster.set(zWrist);
+  }
+ /****************************************************************************************************************************** 
+  ** CONSTRUCTOR
+  ******************************************************************************************************************************/
+  public Wrist(WPI_TalonSRX master) {
+    mMaster = master;
+
+    //ErrorCode errorCode;
+
+
+    // Configure the feedback sensor
+    mMaster.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, RobotMap.kPIDLoopIdx, RobotMap.kLongCANTimeoutMs);
```

Comment:
For all of these Talon setup commands we should be checking their return status

Reply:
Doing the ErrorCode checking, you mean? As in, 
if (ErrorCode != ErrorCode.ok) {
  mLogger.info("Error");
}

Reply:
yes